docs/mobile-apps/automated-testing/espresso-xcuitest/espresso-capture.md

```diff
@@ -0,0 +1,39 @@
+---
+id: espresso-capture
+title: Espresso Screenshot Capture
+sidebar_label: Espresso Screenshot Capture
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The Espresso Screenshot Capture functionality allows you to take screenshots of errors to support the visual verification of your debugging process. These screenshots also help to identify any errors or bugs within your application.
+
+
+## Capturing Screenshots
+
+To take screenshots during an Espresso test run, follow the steps below. Make sure to handle exceptions in your test code to ensure a proper test run. This feature is supported on Android 4.3 and above.
```

Comment:
We don't have 4.3 anywhere, not Emulators and not for RDC. Emulators is 5.1 and RDC is 7.0

Reply:
Corrected.